client/app/components/browse-list/template.hbs

```diff
@@ -4,10 +4,10 @@
             {{#if list}}
                 <div class="col-xs-6">
                     {{#if organizeMode}}
-                        {{#unless groupView}}
-                            <button class="btn btn-default" {{action 'toggleGroupConfirmation'}}>Group Selected</button>
-                        {{/unless}}
-                        <button class="btn btn-danger" {{action 'toggleDeleteConfirmation'}}>Delete selected </button>
+                        {{#if (not groupView)}}
+                            <button class="btn btn-default {{unless selectedItems.length 'disabled'}}" {{action 'toggleGroupConfirmation'}}>Group Selected</button>
```

Comment:
The "Group selected" button appears disabled, but the modal still appears when I click on the button:  
![screen shot 2017-04-07 at 9 46 48 am](https://cloud.githubusercontent.com/assets/6414394/24802797/38287a5e-1b77-11e7-93bb-e9bdffd62540.png)